docs/base/part_properties.rst

```diff
@@ -0,0 +1,372 @@
+.. Ideally, this would be automatically generated.
+
+.. _after:
+
+after
+-----
+**Type:** array of unique strings with at least 1 item
+**Step:** build
+
+Specifies a list of parts that a given part will be built *after*.
+
+.. _build_attributes:
+
+build-attributes
+----------------
+**Type:** array of unique strings with at least 1 item from "core22-step-dependencies", "enable-patchelf", "no-patchelf", "no-install", "debug", "keep-execstack".
```

Comment:
core22-step-dependencies is only relevant to snapcraft; build-attributes as a whole might only be relevant to snapcraft

Reply:
my suggestion is that we address that separately, because if we remove it from this document we won't have anywhere to put it right?

Reply:
we can put it somewhere once we consume this documentation from Snapcraft, which we don't today, this can confuse our rocks users too

Reply:
I've conditionally excluded this for now.